tests/spec/SpecHelper.js

```diff
@@ -179,9 +179,9 @@ function makeDefaultBody() {
  * @returns {{config: {editors, specStatus, edDraftURI, shortName, previousMaturity, previousPublishDate, errata, implementationReportURI, perEnd, lint} & any, body: string}}
  */
 
-function makeStandardOps(configParams, bodyParams = makeDefaultBody()) {
+function makeStandardOps(configParams = {}, bodyParams = makeDefaultBody()) {
```

Comment:
Rename the params “config” and “body”, otherwise it won’t work :)

Reply:
done :smile: